<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackingGui</class>
 <widget class="QMainWindow" name="TrackingGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TrackingGui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="F_Camera">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PB_ConnectCam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_StartCam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_StopCam">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>70</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QFrame" name="TrackingFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>301</width>
       <height>121</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="PB_StartTracking">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Start tracking</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_TrackingThresh">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Label_TrackingThreshold">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Threshold</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label_WindowRadius">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Window Radius </string>
      </property>
     </widget>
     <widget class="QPushButton" name="PB_StopTracking">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Stop tracking</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PB_UseWindowing">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Windowing</string>
      </property>
     </widget>
     <widget class="QPushButton" name="PB_UseThresholding">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>50</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Use Tresholding</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_WindowRadius">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>45</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QComboBox" name="CB_Baumer">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Baumer Bisamberg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Baumer IQOQI</string>
      </property>
     </item>
    </widget>
    <widget class="QFrame" name="CameraFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>301</width>
       <height>211</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="Label_OffsetY">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Offset Y</string>
      </property>
     </widget>
     <widget class="QLabel" name="Label_OffsetX">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Offset X</string>
      </property>
     </widget>
     <widget class="QSlider" name="HS_Width">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Label_ExposureCam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>  Exposure time (us)</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_ExposureCam">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Label_FrameSize">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
     <widget class="QSlider" name="HS_OffsetY">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="HS_OffsetX">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_GainCam">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_Width">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="HS_Height">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Label_Gain">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>  Gain</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_Height">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSlider" name="HS_ExposureCam">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="Label_Height">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>54</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_OffsetY">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>180</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="SB_OffsetX">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>150</y>
        <width>61</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="F_LogCam">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>391</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTextEdit" name="TE_LogCam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>371</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="PB_Disconnect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>500</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Disconnect</string>
    </property>
   </widget>
   <widget class="QFrame" name="HexapodFrame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PB_StartStabilization">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start stabilization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_StopStabilization">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop stabilization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_ConnectHexapod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect hexapod</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_ControlHexapod">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Manually control hexapod</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="SeeingFrame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PB_ConfigureSeeing">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Configure seeing measurement</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_StartSeeingMeasurement">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start seeing measurement</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_StopSeeingMeasurement">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>241</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop seeing measurement</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="HedyLamarrFrame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>391</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="PB_StartHedyLamarrStabilization">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Start stabilization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_StopHedyLamarrStabilization">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop stabilization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_ConnectHedyLamarr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect Hedy Lamarr</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PB_DisconnectHedyLamarr">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Disconnect Hedy Lamarr</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>755</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTracking_GUI">
    <property name="title">
     <string>Tracking GUI</string>
    </property>
   </widget>
   <addaction name="menuTracking_GUI"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
